src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1353,4 +1355,61 @@ private AdsDataProvider() {
                 }
         };
     }
+
+  @DataProvider
+  public static Object[][] adsVelesTracking() {
+    final String queryParamTurnOnVeles = "?InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
+
+    return new Object[][]{
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles" + queryParamTurnOnVeles),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "20.00")
+                .put(AdsContent.INCONTENT_PLAYER, "20.00")
+                .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Incontent" + queryParamTurnOnVeles),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "NOT_INVOLVED")
+                .put(AdsContent.INCONTENT_PLAYER, "20.00")
+                .build()
+        },
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Leaderboard" + queryParamTurnOnVeles),
+            ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "20.00")
+                .put(AdsContent.INCONTENT_PLAYER, "NOT_INVOLVED")
+                .build()
+        }
+    };
+  }
+
+  @DataProvider
+  public static Object[][] adsVelesErrorTracking() {
+    final String queryParamTurnOnVeles = "?InstantGlobals.wgAdDriverVelesBidderCountries=[XX]";
+
+    return new Object[][]{
+      {
+          new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Both/Leaderboard" + queryParamTurnOnVeles),
+          ImmutableMap.builder()
+                .put(AdsContent.TOP_LB, "")
+                .put(AdsContent.INCONTENT_PLAYER, "")
+                .build(),
+          ImmutableMap.builder()
+                .put(".*output=vast.*", new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.REQUEST_TIMEOUT))
+                .build()
+
+      },
+      {
+          new Page(WIKI_SPECIAL, "Project43_Wikia" + queryParamTurnOnVeles), // Veles Timeout (page without VAST)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)